relentless/potential/pair.py

```diff
@@ -589,9 +589,123 @@ class Depletion(PairPotential):
         List of types (A type must be a str).
 
     """
+
+    class LowBound(core.DependentVariable):
+        r"""Dependent variable for the low bound of the depletion pair potential.
+
+        .. math::
+
+            low = \frac{1}{2}(\sigma_i+\sigma_j)
+
+        Parameters
+        ----------
+        sigma_i : int/float or :py:class:`Variable`
+            The `sigma_i` parameter of the low bound.
+        sigma_j : int/float or :py:class:`Variable`
+            The `sigma_j` parameter of the low bound.
+
+        """
+        def __init__(self, sigma_i, sigma_j):
+            super().__init__(sigma_i=sigma_i, sigma_j=sigma_j)
+
+        @property
+        def value(self):
+            return 0.5*(self.sigma_i.value + self.sigma_j.value)
+
+        def _derivative(self, param):
+            """Calculates the derivative of the LowBound object with respect to its parameters.
+
+            Parameters
+            ----------
+            param : str
+                The parameter with respect to which to take the derivative.
+                (Can only be 'sigma_i' or 'sigma_j').
+
+            Returns
+            -------
+            float
+                The calculated derivative value.
+
+            Raises
+            ------
+            ValueError
+                If the parameter argument is not 'sigma_i' or 'sigma_j'.
+
+            """
+            if param == 'sigma_i':
+                return 0.5
+            elif param == 'sigma_j':
+                return 0.5
+            else:
+                raise ValueError('Unknown parameter')
+
+    class HighBound(core.DependentVariable):
```

Comment:
Let's rename this `Cutoff` to make it a little more self-explanatory. the user would call it like `Depletion.Cutoff`